packages/mobile/src/components/FeeDrawer.tsx

```diff
@@ -87,21 +88,22 @@ export default function FeeDrawer({
       currencyCode: CURRENCIES[currency].code,
     }
 
-  return (
+  return hasNonZeroFees ? (
     // Uses View instead of Fragment to workaround a glitch with LayoutAnimation
     <View>
       <Touchable onPress={toggleExpanded} testID={testID}>
         <View style={styles.totalContainer}>
-          <Expandable isExpandable={true} isExpanded={expanded}>
+          <Expandable isExpandable={hasNonZeroFees} isExpanded={expanded}>
```

Comment:
won't `isExpandable` always be true since this component is conditionally rendered?

Reply:
Yes! This was either a style point I was unsure about or a relic of earlier work before I changed the render itself to have a condition. This should be reverted to `isExpandable={true}` to trust the earlier condition :)